Mathlib/NumberTheory/NumberField/InfinitePlace/Ramification.lean

```diff
@@ -342,17 +342,17 @@ lemma card_stabilizer [IsGalois k K] :
   · rwa [← not_isUnramified_iff_card_stabilizer_eq_two]
 
 lemma even_nat_card_aut_of_not_isUnramified [IsGalois k K] (hw : ¬ IsUnramified k w) :
-    Even (Nat.card <| K ≃ₐ[k] K) := by
-  by_cases H : Finite (K ≃ₐ[k] K)
-  · cases nonempty_fintype (K ≃ₐ[k] K)
+    Even (Nat.card <| Gal(K/k)) := by
```

Comment:
Can you please remove `<|` before `Gal` in this file (and elsewhere, if I've missed it)? (Find-replace for `<| Gal` should work.)